.github/workflows/build-and-release-clients.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and release OneLife client
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Please use the tag base `2HOL_v*` as sometimes other tags are used on this repository that should not trigger a build.